gnmpagerduty/management/commands/install_pagerduty_key.py

```diff
@@ -0,0 +1,25 @@
+from django.core.management.base import BaseCommand
```

Comment:
Can this not be put into the install_pagerduty command? Seems odd to have two commands and two configuration blocks